### ISSUE TITLE
Missing title attribute on iframe in CmsGdprVideoElement (accessibility issue)

### ISSUE THREAD

**Issue by paslem153** [shopware/platform#11291]
### Shopware Version

6.6.10.5

### Affected area / extension

Platform(Default)

### Actual behaviour

When `config.needsConfirmation.value` is set to true, the CmsGdprVideoElement plugin dynamically injects an `<iframe>` via JavaScript after user consent. However, the resulting `<iframe>` lacks a title attribute.

This violates accessibility standards (e.g. WCAG 2.1), as assistive technologies cannot identify the content or purpose of the iframe.

In contrast, if `needsConfirmation` is set to false, the iframe is rendered directly via the twig template and does include a proper title attribute.

**Edit**: As far as I can see, the iframeTitle is considered in the function `_replaceElementWithVideo()`, but not registered in the static options

### Expected behaviour

The dynamically created `<iframe>` should also include a descriptive and accessible title attribute.
Ideally, this title should be the configurable variable that is also used in the Twig template: `{{ config.iframeTitle.value }}`

### How to reproduce

1. Create a CMS YouTube element in a Shopping Experience.
2. Ensure config.needsConfirmation.value is set to true.
3. Open the storefront and accept the video consent.
4. Inspect the injected <iframe> in the DOM.
5. Observe that the title attribute is missing.

Compare with the case when needsConfirmation is set to false, which renders the following server-side:
```
<iframe class="{{ iframeClass }}"
        src="{{ videoUrl }}"
        title="{{ config.iframeTitle.value }}"
        allowfullscreen="allowfullscreen">
</iframe>
```

### Definition of Done

- [ ] An automated test covers the fix.
- [ ] Integration/E2E testing in staging is done.
- [ ] Changelog markdown file created or updated in /changelog/_unreleased directory.
- [ ] Developer documentation written/updated.
- [ ] End user and/or developer documentation written or updated with clarification.
- [ ] Regression test created to ensure bug does not reappear, and automated regression tests pass.
- [ ] Code changes resolve the reported bug without introducing new issues.

**Comment by ravenokko** [comment]
Missing backport: https://github.com/shopware/shopware/pull/7636
